src/test/memory-leak/runMemlabTests.js

```diff
@@ -0,0 +1,15 @@
+const { run } = require('@memlab/api');
+
+const tests = ['./tests/toggleMobileMenu', './tests/sliderScroll'];
+
+// TODO: make all tests run by one command
+(async function () {
```

Comment:
looks like hardcode, need to investigate another approach to implement, and need to try use ts, not js

Reply:
not fixed

Reply:
Memlab does not have the capability to run test suites like Jest, and I'm not entirely sure where the hardcoding is if we dynamically take all tests from the folder. Also, Memlab does not support TypeScript.

Reply:
do you investigate another approach about how we can execute all tests?

Reply:
Yes, I have been looking for better solutions but haven't found any. If you have any ideas on this, please share.